airbyte-integrations/connectors/source-railz/source_railz/manifest.yaml

```diff
@@ -0,0 +1,362 @@
+    stream_cursor_field: "postedDate"
+    primary_key: "id"
+  invoices_credit_notes_stream:
+    $ref: "*ref(definitions.base_incremental_service_stream)"
+    $options:
+      name: "invoices_credit_notes"
+      path: "/invoices/creditNotes"
+      cursor_field: "postedDate"
+      serviceNames: ["freshbooks", "quickbooks", "xero", "oracleNetsuite", "sageIntacct", "dynamicsBusinessCentral", "myob"]
+    stream_cursor_field: "postedDate"
+    primary_key: "id"
+  invoices_payments_stream:
+    $ref: "*ref(definitions.base_incremental_service_stream)"
+    $options:
+      name: "invoices_payments"
+      path: "/invoices/payments"
+      cursor_field: "date"
+      serviceNames: ["freshbooks", "quickbooks", "quickbooksDesktop", "xero", "oracleNetsuite", "sageBusinessCloud", "sageIntacct", "dynamicsBusinessCentral", "myob"]
+    stream_cursor_field: "date"
+    primary_key: "id"
+  journal_entries_stream:
+    $ref: "*ref(definitions.base_incremental_service_stream)"
+    $options:
+      name: "journal_entries"
+      path: "/journalEntries"
+      cursor_field: "postedDate"
+      serviceNames: ["freshbooks", "quickbooks", "quickbooksDesktop", "xero", "oracleNetsuite", "sageBusinessCloud", "sageIntacct", "myob"]
+    stream_cursor_field: "postedDate"
+    primary_key: "id"
+  purchase_orders_stream:
+    $ref: "*ref(definitions.base_incremental_service_stream)"
+    $options:
+      name: "purchase_orders"
+      path: "/purchaseOrders"
+      cursor_field: "postedDate"
+      serviceNames: ["quickbooks", "xero", "oracleNetsuite", "sageIntacct", "dynamicsBusinessCentral", "myob"]
+    stream_cursor_field: "postedDate"
+    primary_key: "id"
+  refunds_stream:
+    $ref: "*ref(definitions.base_incremental_service_stream)"
+    $options:
+      name: "refunds"
+      path: "/refunds"
+      cursor_field: "date"
+      serviceNames: ["quickbooks", "xero", "myob"]
+    stream_cursor_field: "date"
+    primary_key: "id"
+  commerce_disputes_stream:
+    $ref: "*ref(definitions.base_incremental_service_stream)"
+    $options:
+      name: "commerce_disputes"
+      path: "/commerce/disputes"
+      cursor_field: "createdDate"
+      serviceNames: ["shopify", "square", "myob"]
+    stream_cursor_field: "createdDate"
+    primary_key: "id"
+  commerce_orders_stream:
+    $ref: "*ref(definitions.base_incremental_service_stream)"
+    $options:
+      name: "commerce_orders"
+      path: "/commerce/orders"
+      cursor_field: "createdDate"
+      serviceNames: ["shopify", "square", "myob"]
+    stream_cursor_field: "createdDate"
+    primary_key: "id"
+  commerce_products_stream:
+    $ref: "*ref(definitions.base_incremental_service_stream)"
+    $options:
+      name: "commerce_products"
+      path: "/commerce/products"
+      cursor_field: "createdDate"
+      serviceNames: ["shopify", "square", "myob"]
```

Comment:
[the docs](https://docs.railz.ai/reference/product) don't have `myob` as a valid option, do you know why this is the case?

Reply:
In really I checked this list empirical way. For every stream, I looped over all possible service names and check the status code. For some endpoints, I could find working serviceNames that were not specified in the documentation.